keps/2724-topology-aware-scheduling/README.md

```diff
@@ -399,6 +409,46 @@ is that each Job will be placed within a "host". It is also possible
 to omit the annotation `kueue.x-k8s.io/podset-slice-size`, as the default
 value for it in case of JobSet is `parallelism`.
 
+##### LeaderWorkerSet
+
+According to [Story 6](#story-6) we noticed that some users would like to
+co-locate Leader with Workers within the same replica in LeaderWorkerSet.
+
+To allow user to specify the requested topology for leader and workers we 
+are adding an annotation `kueue.x-k8s.io/leaderworkerset-group-required-topology`.
+
+**Example**:
+
+```yaml
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  name: tas-example-lws
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+  annotations:
+    kueue.x-k8s.io/leaderworkerset-group-required-topology: "cloud.provider.com/topology-rack"
```

Comment:
Just for confirmation about LWS. Are these top-level annotations and labels propagated to the underlying Pods?